readme.md

```diff
@@ -26,7 +26,7 @@ intoStream('unicorn').pipe(process.stdout);
 
 ### intoStream(input)
 
-Type: `Buffer` `string` `Iterable<Buffer|string>` `Promise`<br>
+Type: `Buffer` `TypedArray` `string` `Iterable<Buffer|string>` `Promise`<br>
```

Comment:
This should mention `ArrayBuffer` too, as TypedArray doesn't include ArrayBuffer. ArrayBuffer is the backing-store of a TypedArray.